# 0x05. Python - Exceptions

This project folder cntains programs that ilustrate an elementary introduction to the try/except/finally methods in Python.

## Files Description

A brief description of what is contained inside the files:

---

### Task 0

- File: 0-safe_print_list.py
- A function that prints x elements of a list
- Prototype:

```python
def safe_print_list(my_list=[], x=0):
```

### Task 1

- File: 1-safe_print_integer.py
- A function that prints an integer with "{:d}".format()
- Prototype:

```python
def safe_print_integer(value):
```

### Task 2

- File: 2-safe_print_list_integers.py
- A function that prints the first x elements of a list and only integers
- Prototype:

```python
def safe_print_list_integers(my_list=[], x=0):
```

### Task 3

- File: 3-safe_print_division.py
- A function that divides 2 integers and prints the result
- Prototype:

```python
def safe_print_division(a, b):
```

### Task 4

- File: 4-list_division.py
- A function that divides element by element 2 lists
- Prototype:

```python
def list_division(my_list_1, my_list_2, list_length):
```

### Task 5

- File: 5-raise_exception.py
- A function that raises a type exception
- Prototype:

```python
def raise_exception():
```

### Task 6

- File: 6-raise_exception_msg.py
- A function that raises a name exception with a message
- Prototype:

```python
def raise_exception_msg(message=""):
```

### Task 

- File:
- A 
- Prototype:

```python

```
### Task 

- File:
- A 
- Prototype:

```python

```